backend/controllers/posts.controller.js

```diff
@@ -58,4 +58,56 @@ exports.getOtherUsersPosts = async (req, res) =>{
     } catch (error) {
         return res.status(500).send({message: `Server error: ${error.message}`})
     }
+}
+
+exports.deletePost = async (req, res) =>{
+    try{
+        const postId = req.params.id;
+        const postBy = Post.findById(postId).postBy;
```

Comment:
missing await keyword here

Reply:
separate actions of getting the post from database and then in another variable save the postedBy property, then do the comparison